src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -1236,6 +1236,54 @@ namespace cryptonote
     typedef epee::misc_utils::struct_init<response_t> response;
   };
 
+  struct public_node
+  {
+    std::string host;
+    uint64_t last_seen;
```

Comment:
You might want to zero this is restricted RPC.

Reply:
Have some concerns about it being used for fingerprinting.
Currently `get_public_nodes` works only in non-restricted mode only (https://github.com/monero-project/monero/pull/5607/files#diff-8238353d591b683ea88296ba92b7ee25R111)
For example, `get_peer_list` acts the same as well.

Any thoughts?

Reply:
Oh good, it's fine then.